packages/apps/human-app/server/src/app.module.ts

```diff
@@ -21,9 +25,11 @@ import { envValidator } from './common/config/env';
     }),
     HttpModule,
     WorkerModule,
-    IntegrationsModule,
+    OperatorModule,
+    ReputationOracleModule,
+    AppConfigModule,
```

Comment:
Why AppConfigModule is created? We can pass config services (GatewayConfigService, EnvironmentConfigService) to Providers inside app.module.ts. imo should only be one application configuration file.